scripts/test/run-canary-test.sh

```diff
@@ -115,7 +117,7 @@ function set_env_aws_node() {
 
 function run_canary_tests() {
   (cd $INTEGRATION_TEST_DIR/perpodsg && CGO_ENABLED=0 ginkgo --focus="CANARY" -v -timeout 15m -- -cluster-kubeconfig=$KUBE_CONFIG_PATH -cluster-name=$CLUSTER_NAME --aws-region=$REGION --aws-vpc-id $VPC_ID)
-  (cd $INTEGRATION_TEST_DIR/windows && CGO_ENABLED=0 ginkgo --focus="CANARY" -v -timeout 25m -- -cluster-kubeconfig=$KUBE_CONFIG_PATH -cluster-name=$CLUSTER_NAME --aws-region=$REGION --aws-vpc-id $VPC_ID)
+  (cd $INTEGRATION_TEST_DIR/windows && CGO_ENABLED=0 ginkgo --focus="CANARY" -v -timeout 27m -- -cluster-kubeconfig=$KUBE_CONFIG_PATH -cluster-name=$CLUSTER_NAME --aws-region=$REGION --aws-vpc-id $VPC_ID)
```

Comment:
Curious...How did we arrive at these numbers? I do see that we already have hard coded/specific timeout values for these tests but a comment on how we converged at these numbers will help. Even better if they are configurable but that can be a separate enhancement.

Reply:
If 25m timeout did not work, 27m would likely not work for the same reason. Please explain why 27m is a better choice here.